olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1366,11 +1366,11 @@ spec:
         - knativeeventings.operator.knative.dev
   relatedImages:
     - name: "knative-operator"
-      image: "registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-rhel8-operator@sha256:f0137119145df3719e6f6f423b78adfea198fe08cd7d2af67bbbe7eae016d0ca"
+      image: "registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-operator:latest-rhel8@serverless-openshift-kn-operator:latest"
```

Comment:
This looks weird
Put /hold for now.

Reply:
/hold

Reply:
How is that name change happening?

Isn't that generated by scripts?

Reply:
Maybe the issue is somewhere here, as we have for some image names some corner cases: https://github.com/openshift-knative/serverless-operator/blob/d5e616122fe9300efca15f5f76c27beb4a7d5c57/hack/lib/images.bash#L419-L476